static/js/theme-map/ThemeMap.js

```diff
@@ -139,7 +139,10 @@ class ThemeMap extends ANSWERS.Component {
    */
   addMapInteractions(map) {
     this.map.idle().then(() => {
-      map.setPanHandler(() => this.updateMapPropertiesInStorage());
+      map.setPanHandler((prevousBounds, currentBounds, zoomTrigger) => {
+        this.updateMapPropertiesInStorage();
+        this.config.panHandler(prevousBounds, currentBounds, zoomTrigger);
```

Comment:
Is the handler fired on the end of panning?

Reply:
For MapBox it isn't fired until the map has stopped moving completely. On Google Maps it gets called as the map moves.